src/endpoint.cpp

```diff
@@ -508,7 +507,7 @@ UartEndpoint::~UartEndpoint()
 
 int UartEndpoint::set_speed(speed_t baudrate)
 {
-    struct termios2 tc;
+    struct termios2 tc = {};
```

Comment:
we do a bzero just below. Initializing structs like this actually has a problem: if the struct has padding, the padding is not initialized, so when passed to a syscall it is flagged as "passing garbage to a syscall" by valgrind

Reply:
Fixed for all `struct termios2 tc` and added a comment about the padding issue.